src/thinkspace/client/thinkspace-builder-rat/addon/components/confirmation/details.coffee

```diff
@@ -0,0 +1,29 @@
+import ember from 'ember'
+import base  from 'thinkspace-base/components/base'
+
+###
+# # details.coffee
+- Type: **Component**
+- Package: **ethinkspace-builder-pe**
+###
+export default base.extend
+  builder: ember.inject.service()
+
+  model: ember.computed.reads 'builder.model'
+  step:  ember.computed.reads 'builder.step_details'
+
+  init_base: ->
+    @get('step').validate().then =>
+      @set('changeset', @get('step.changeset'))
+
+  has_title_error: ember.computed 'changeset', ->
```

Comment:
After some more testing, I don't think we're able to do it this way. I do really dislike that computed property though...